dist/init/linux-systemd/README.md

```diff
@@ -34,23 +161,40 @@ Give the caddy binary the ability to bind to privileged ports (e.g. 80, 443) as
 sudo setcap 'cap_net_bind_service=+ep' /usr/local/bin/caddy
 ```
 
+### Working with the upload feature
+
+You may need to comment the lines shown above these lines and uncomment these:
+
+```
+# Comment the section right above this section and uncomment this one
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE CAP_LEASE
+AmbientCapabilities=CAP_NET_BIND_SERVICE CAP_LEASE
+NoNewPrivileges=true
+```
+
+## caddy command not found
+
+```bash
+sudo cp /path/to/caddy /usr/local/bin
+sudo chown root:root /usr/local/bin/caddy
+sudo chmod 755 /usr/local/bin/caddy
+```
+
+### no user or group www-data
+
 Set up the user, group, and directories that will be needed:
 
 ```bash
 sudo groupadd -g 33 www-data
 sudo useradd \
   -g www-data --no-user-group \
-  --home-dir /var/www --no-create-home \
+  --home-dir /srv/www --no-create-home \
```

Comment:
Don't start any **srv vs var** fight here. ;-)

Retain to change the changeset minimal.